README.md

```diff
@@ -83,6 +83,17 @@ Then, start the **Trappist** playground with:
 ./zombienet-linux -p native spawn xcm-playground.toml
 ```
 
+### XCM Simulator
+The [XCM simulator](./xcm-simulator) can be used to further explore XCM message execution across the various runtimes used by Trappist. 
+Each Trappist use case is written as a test, allowing interactive debugging/exploration of message flows and instruction execution.
+Each `execute_with` closure scope within a test can be considered as a block on the corresponding chain, with messages being dispatched to the destination chains via a mock message queue as the closure goes out of scope.
+All XCM-specific traces from the interactions are also collected in a single place for easier inspection. 
+
+You can run all tests, including the tracing output, with:
+```
+cd xcm-simulator && cargo nextest run tests:: --no-capture --release
```

Comment:
maybe add some contextual note about `cargo-nextest`?

or just instruct audience to use vanilla `cargo test`

Reply:
Sorry, that was a copy-paste error and shouldn't have been pushed yet. Think it got caught up in the CI push I did to test that on GitHub. Will get it corrected to standard `cargo test`.

Thanks!